feature-libs/checkout/b2b/root/config/default-b2b-occ-config.ts

```diff
@@ -44,6 +42,8 @@ export const defaultB2bOccConfig: OccConfig = {
           'users/${userId}/replenishmentOrders/${replenishmentOrderCode}?fields=FULL,costCenter(FULL),purchaseOrderNumber,paymentType,user',
         replenishmentOrderHistory:
```

Comment:
shouldn't these things be in the replenishment sub lib entrypoint

Reply:
putting things here vs putting things in /occ/config?

Reply:
Yes, correct. It's already moved in repl PR.